src/css/custom.css

```diff
@@ -123,4 +123,36 @@ td {
   .header-github-link {
     margin-right: 2rem;
   }
+}
+
+.footer__links {
+  justify-content: space-between;
+  width: 100%;
+}
+
+.footer__col {
+  flex: none;
+  width: fit-content;
+}
+
+.footer__items {
+  display: flex;
+  flex-direction: row;
+  gap: 24px;
+  width: fit-content;
+}
+
+.menu__link {
+  font-size: 14px;
+  line-height: 24px;
+}
```

Comment:
The font size in menu is appropriate. We should modify the font size for the documentation content to make it uniform as currently it looks weird.